src/v2/System/useFeatureFlag.tsx

```diff
@@ -1,7 +1,5 @@
 import { useSystemContext } from "v2/System"
 
-export type EnabledFeatureFlags = Record<string, boolean>
```

Comment:
it may be worth moving this type to `v2/Typings/common.ts` (to avoid cyclical imports) and use it [here](https://github.com/artsy/force/blob/5fa3a5cac134c666611ec135f138d18ff76f0cdd/src/desktop/lib/buildClientAppContext.ts#L10) and [here](https://github.com/artsy/force/blob/5fa3a5cac134c666611ec135f138d18ff76f0cdd/src/v2/System/SystemContext.tsx#L82)?